controllers/activegate/token_parser.go

```diff
@@ -43,7 +44,7 @@ func verifySecret(secret *corev1.Secret) error {
 		dtclient.DynatracePaasToken} {
 		_, err := ExtractToken(secret, token)
 		if err != nil {
-			return fmt.Errorf("invalid secret %s, %s", secret.Name, err)
+			return errors.Errorf("invalid secret %s, %s", secret.Name, err)
```

Comment:
Don't we need errors.WithStack here?

Reply:
basically every method in the pkg/errors package adds a stacktrace [https://github.com/pkg/errors/blob/master/errors.go](https://github.com/pkg/errors/blob/master/errors.go)